tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -452,17 +455,19 @@ def floor_return_type_real(x):
         a = floor(x)
         return a
 
-    f1 = epyccel(floor_return_type_int)
+    f1 = epyccel(floor_return_type_int, language = language)
     x = randint(100)
     assert(isclose(f1(x), floor_return_type_int(x), rtol=1e-15, atol=1e-15))
     assert(type(f1(x)) == type(floor_return_type_int(x).item())) # pylint: disable=unidiomatic-typecheck
 
-    f1 = epyccel(floor_return_type_real)
+    f1 = epyccel(floor_return_type_real, language = language)
     x = randint(100)
     assert(isclose(f1(x), floor_return_type_real(x), rtol=1e-15, atol=1e-15))
     assert(type(f1(x)) == type(floor_return_type_real(x).item())) # pylint: disable=unidiomatic-typecheck
 
-def test_shape_indexed():
+def test_shape_indexed(language):
+    if (language == 'c'):
+        pytest.xfail(reason="arrays not implimented yet")
```

Comment:
In other files the array tests have not been activated while they are not yet implemented. You can either remove this and we will add the language option back once arrays are implemented or you need to fix the typo and correct the use of xfail